component/Cards/styles.css

```diff
@@ -41,18 +34,14 @@
 .card-content-txt-color {
     color: gray;
```

Comment:
use variables, as you are using variables for other colors.

Reply:
done